Sources/Workspace/Workspace.swift

```diff
@@ -463,12 +476,14 @@ public class Workspace {
     ///   - fileSystem: The file system to use, defaults to local file system.
     ///   - forRootPackage: The path for the root package.
     ///   - customManifestLoader: A custom manifest loader.
+    ///   - strictFingerprintChecking: Indicates if fingerprint mismatch would result in warnings or failures.
     ///   - delegate: Delegate for workspace events
     public convenience init(
         fileSystem: FileSystem? = .none,
         forRootPackage packagePath: AbsolutePath,
-        customManifestLoader: ManifestLoaderProtocol? =  .none,
-        delegate: WorkspaceDelegate? =  .none
+        customManifestLoader: ManifestLoaderProtocol? = .none,
+        strictFingerprintChecking: Bool? = .none,
```

Comment:
for this initializer I think we do not need to add this option - similar to other resolver options that use the default